examples/machine-learning/a3-megagpu-8g/slurm-a3mega-cluster.yaml

```diff
@@ -215,6 +215,10 @@ deployment_groups:
           chmod 0755 "${SLURM_ROOT}/scripts/rxdm"
           ln -s "${SLURM_ROOT}/scripts/rxdm" "${SLURM_ROOT}/partition-$(vars.a3mega_partition_name)-prolog_slurmd.d/rxdm.prolog_slurmd"
           ln -s "${SLURM_ROOT}/scripts/rxdm" "${SLURM_ROOT}/partition-$(vars.a3mega_partition_name)-epilog_slurmd.d/rxdm.epilog_slurmd"
+          cp community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/tools/gpu-test "${SLURM_ROOT}/scripts/gpu-test"
```

Comment:
The file will be already present on VMs, copied from gSC as part of tarball with all python scripts. 
No need for `cp` & `chmod`. 
You can find it in `/slurm/scripts/tools/gpu-test`

Reply:
Ah, I see. I will remove cp and chmod. For my own education, how and when does the script become available under `/slurm/scripts/tools/gpu-test`? Could you share with me the code link? Thanks!